4-structural-pseudo-class.json

```diff
@@ -4,24 +4,24 @@
     "name": "Structural pseudo-class",
     "description": "The pseudo-class **structural** introduces the `:nth-child()` and `:nth-last-child()` selector which is similar to the [`:nth-child()`](https://css4-selectors.com/selector/css3/structural-pseudo-class/#nth-child 'CSS3: Structural Pseudo Class') in CSS3. The difference is that this selector will match the given selector list composed of their [inclusive siblings](https://dom.spec.whatwg.org/#concept-tree-inclusive-sibling 'WHATWG: DOM Specification').",
     "statistics_regex": ":nth-child\\s*\\((.*)\\s+of\\s+(.*)\\)",
-    "example": ":nth-child()",
+    "example": ":nth-child(1 of :not([pinned], [hidden]))",
```

Comment:
Please use generalised `1 of s1, s2, …` instead to match the other selector examples. It's just used in the overview page.